src/sagemaker/serve/model_server/hf_dlc/prepare.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Prepare HF DLC Model for Deployment"""
+
+from __future__ import absolute_import
+import logging
+from pathlib import Path
+
+from sagemaker.serve.utils.local_hardware import _check_disk_space, _check_docker_disk_usage
+
+logger = logging.getLogger(__name__)
+
+
+def _create_dir_structure(model_path: str) -> tuple:
```

Comment:
I don't really get why we can't put this into a utility file. This function is already duplicated twice

Reply:
Agree, but I think it should be out of scope of this PR and taken as a general code refactor

Reply:
Moved to a separate file